server/service/src/vaccine_course/test/query.rs

```diff
@@ -1,6 +1,6 @@
 #[cfg(test)]
 mod query {
-    use repository::mock::{mock_demographic_indicator_a, mock_program_a, MockDataInserts};
+    use repository::mock::{mock_immunisation_program, MockDataInserts};
```

Comment:
small side track, but figured we should be using an immunisation program in our tests...